app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -42,13 +42,24 @@ def create_usps_enrollment(enrollment, pii)
         # Use the enrollment's unique_id value if it exists, otherwise use the deprecated
         # #usps_unique_id value in order to remain backwards-compatible. LG-7024 will remove this
         unique_id = enrollment.unique_id || enrollment.usps_unique_id
-        address = [pii['address1'], pii['address2']].select(&:present?).join(' ')
+
+        # Combine address parts and convert to help pass USPS API validation
+        address = I18n.transliterate(
+          [pii['address1'], pii['address2']].select(&:present?).join(' '),
+          locale: :en,
+        ).gsub(/[^A-Za-z0-9\-' .\/#]/, '').gsub(/\s+/, ' ').strip
+
+        # Convert first and last names to help them pass USPS API validation
+        first_name = I18n.transliterate(pii['first_name'], locale: :en).
+          gsub(/[^A-Za-z\-' ]/, '').gsub(/\s+/, ' ').strip
+        last_name = I18n.transliterate(pii['last_name'], locale: :en).
+          gsub(/[^A-Za-z\-' ]/, '').gsub(/\s+/, ' ').strip
```

Comment:
Couple questions:

1. Is it a safe assumption that a name wouldn't include a number? (Those "falsehoods programmers believe about X" type articles came to mind, and I see this specific assumption included on [one such list about names](https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/)).
2. Should we create a utility method(s) for the reused code here, to DRY it up and make it clearer what we're accomplishing (via method naming)?

Reply:
>Is it a safe assumption that a name wouldn't include a number?

I guess there's an argument to be made that we're intentionally losing some accuracy with the names to conform to the APIs requirements, but I don't know that stripping numbers is a requirement, since at least the first name field is documented as accepting an "alphanumeric" value (`lastName`, on the other hand, may need to be formatted 😩 ).

Reply:
I tested [and verified previously](https://gsa-tts.slack.com/archives/C03FA4VBN76/p1659128315140999?thread_ts=1659041747.865099&cid=C03FA4VBN76) that the `firstName` field also rejects numbers.

Reply:
> 2. Should we create a utility method(s) for the reused code here, to DRY it up and make it clearer what we're accomplishing (via method naming)?

I think we should defer abstracting this until another use case comes up.